airbyte-integrations/bases/base-python/base_python/schema_helpers.py

```diff
@@ -38,8 +38,7 @@ def __init__(self, shared_schemas_path: str):
 
     @staticmethod
     def _load_shared_schema_refs(shared_schemas_path: str):
-        shared_file_names = [f for f in os.scandir(shared_schemas_path) if f.isfile()]
-
+        shared_file_names = [f.name for f in os.scandir(shared_schemas_path) if f.is_file()]
```

Comment:
Any way you could have some unit tests for these?

Reply:
yes, top priority after refactoring the package